.github/workflows/issue-management.yml

```diff
@@ -0,0 +1,87 @@
+name: Issue Management
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # runs daily at midnight UTC
+  workflow_dispatch: # allows you to manually trigger the action
+
+jobs:
+  manage-issues:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Manage inactive issues
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
```

Comment:
Nit: Is it possible to move all of this script logic to a separate JS file and import that JS file here? Thinking we could put this in like a `.github/scripts` folder. Feel like that could make it easier to update/maintain since the formatting in a `.yml` file is the equivalent of a long string haha.